include/alpaka/queue/QueueOaccBlocking.hpp

```diff
@@ -0,0 +1,42 @@
+/* Copyright 2019 Benjamin Worpitz, René Widera
+ *
+ * This file is part of Alpaka.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+#pragma once
+
+#ifdef ALPAKA_ACC_ANY_BT_OACC_ENABLED
+
+#if _OPENACC < 201306
+    #error If ALPAKA_ACC_ANY_BT_OACC_ENABLED is set, the compiler has to support OpenACC xx or higher!
+#endif
+
+#include <alpaka/dev/DevOacc.hpp>
+
+#include <alpaka/dev/Traits.hpp>
+#include <alpaka/event/Traits.hpp>
+#include <alpaka/queue/Traits.hpp>
+#include <alpaka/wait/Traits.hpp>
+
+#include <alpaka/queue/QueueGenericThreadsBlocking.hpp>
+
+#include <stdexcept>
+#include <memory>
+#include <functional>
+#include <mutex>
+#include <condition_variable>
+#include <thread>
+
+namespace alpaka
+{
+    namespace queue
+    {
+        using QueueOaccBlocking = QueueGenericThreadsBlocking<dev::DevOacc>;
+    }
+}
```

Comment:
Feels like too many includes for this one alias definition ;)

Reply:
True.